src/main/java/uk/gov/justice/hmpps/prison/service/enteringandleaving/ReleasePrisonerService.kt

```diff
@@ -35,9 +40,10 @@ class ReleasePrisonerService(
     bedAssignmentMovementService.endBedHistory(booking.bookingId, movement.movementTime)
     sentenceMovementService.deactivateSentences(booking.bookingId)
     paymentsMovementService.endPaymentRules(booking.bookingId, movement.movementDate)
-    activityMovementService.endActivitiesAndWaitlist(booking, toLocation, movement.movementDate, request.movementReasonCode)
+    activityMovementService.endActivitiesAndWaitlist(booking, movement.fromAgency, movement.movementDate, request.movementReasonCode)
```

Comment:
Bug that relies on the integration tests found when switching to the new service.